Scripts/install.sh

```diff
@@ -1,48 +1,26 @@
 #!/bin/bash
 
-lower () {
-    IFS='.' read -a v1_array <<< "$1"
-    IFS='.' read -a v2_array <<< "$2"
-    v1=$((v1_array[0] * 100 + v1_array[1] * 10 + v1_array[2]))
-    v2=$((v2_array[0] * 100 + v2_array[1] * 10 + v2_array[2]))
-    diff=$((v2 - v1))
-
-    ((diff >  0)) && return 0
-    return 1
-}
-
 TOOL_NAME="danger-swift"
 PREFIX=${PREFIX:='/usr/local'}
 INSTALL_PATH="$PREFIX/bin/$TOOL_NAME"
-BUILD_PATH=".build/release/$TOOL_NAME"
 LIB_INSTALL_PATH="$PREFIX/lib/danger"
 declare -a SWIFT_LIB_FILES=('libDanger.dylib' 'libDanger.so' 'Danger.swiftdoc' 'Danger.swiftmodule' 'OctoKit.swiftdoc' 'OctoKit.swiftmodule' 'RequestKit.swiftdoc' 'RequestKit.swiftmodule' 'Logger.swiftdoc' 'Logger.swiftmodule' 'DangerShellExecutor.swiftdoc' 'DangerShellExecutor.swiftmodule')
 
 
 DANGER_LIB_DECLARATION='\.library(name:\ \"Danger\", targets: \[\"Danger\"\])'
 DANGER_LIB_DYNAMIC_DECLARATION='\.library(name:\ \"Danger\",\ type:\ \.dynamic,\ targets:\ \[\"Danger\"\])'
-sed  "s/$DANGER_LIB_DECLARATION/$DANGER_LIB_DYNAMIC_DECLARATION/g" Package.swift > tmpPackage
+sed "s/$DANGER_LIB_DECLARATION/$DANGER_LIB_DYNAMIC_DECLARATION/g" Package.swift > tmpPackage
 mv -f tmpPackage Package.swift
 
 swift package clean
 
-SWIFT_VERSION=`swift -version | head -n 1 | perl -lpe 's/.*version\ (\d\.\d\.\d).*/$1/'`
-
-UNAME_OUT="$(uname -s)"
-
-lower $SWIFT_VERSION '5.3.0'
+BUILD_FOLDER=".build/release"
+swift build --disable-sandbox -c release
 
-if [[ "$?" -eq 0 || "$OSTYPE" == "darwin"* ]]
-then
-    BUILD_FOLDER=".build/release"
-    if [[ `uname -m` == 'arm64' ]]; then
-        swift build --disable-sandbox --arch arm64 -c release
-    else
-        swift build --disable-sandbox -c release
-    fi
-else
+if [ $? -ne 0 ]; then
+    echo '[WARN] Failed to install with `release` configuration. Try to install with `debug` configuration.'
     BUILD_FOLDER=".build/debug"
-    swift build --disable-sandbox
+    swift build --disable-sandbox -c debug
```

Comment:
[`make install` was failed in 5.5.3](https://github.com/danger/swift/runs/6988748337?check_suite_focus=true) so workaround should be still alive 🙄
However, [`make install` in 5.4.3 succeeds](https://github.com/danger/swift/runs/6988748235?check_suite_focus=true) and it should not switch configuration by Swift version.